makefile

```diff
@@ -1,8 +1,9 @@
 .DEFAULT_GOAL := help
 
 GOLLUM_TAG := $(shell git describe --always --tags --match "v*" | sed -E 's/^v([0-9\.]+)(-[0-9]+){0,1}((-)g([a-f0-9]+)){0,1}.*/\1\2\4\5/')
+GOLLUM_RELEASE_SUFFIX := # Can be set for custom releases to also include plugin versions
 GOLLUM_DIRTY := $(if $(shell git status --porcelain),-dirty)
-GOLLUM_VERSION := $(join $(GOLLUM_TAG),$(GOLLUM_DIRTY))
+GOLLUM_VERSION := $(GOLLUM_TAG)$(GOLLUM_DIRTY)$(if $(GOLLUM_RELEASE_SUFFIX),-$(GOLLUM_RELEASE_SUFFIX))
```

Comment:
To keep the version string semver compatible, we should treat the suffix as metadata, see https://semver.org/spec/v2.0.0.html#spec-item-10 .
In this case it should be `+$(GOLLUM_RELEASE_SUFFIX)`.